poc_mobile_verifier/src/verifier.rs

```diff
@@ -150,6 +127,45 @@ impl VerifierDaemon {
 
         Ok(())
     }
+
+    async fn last_verified_end_time(&self) -> Result<DateTime<Utc>> {
+        Ok(Utc.timestamp(meta::get(&self.pool, "last_verified_end_time").await?, 0))
+    }
+
+    async fn save_last_verified_end_time(
+        exec: impl PgExecutor<'_>,
+        value: &DateTime<Utc>,
+    ) -> Result {
+        meta::save(exec, "last_verified_end_time", value.timestamp() as i64)
+            .await
+            .map_err(Error::from)
+    }
+
+    async fn last_rewarded_end_time(&self) -> Result<DateTime<Utc>> {
+        Ok(Utc.timestamp(meta::get(&self.pool, "last_rewarded_end_time").await?, 0))
+    }
+
+    async fn save_last_rewarded_end_time(
```

Comment:
You shouldn't scope functions to a struct that don't either take `Self` as an argument or return a type that includes `Self` - these should all be free functions

Reply:
Good cal, Fixed ebaa599